target/linux/ipq40xx/base-files/lib/preinit/05_set_iface_mac_ipq40xx.sh

```diff
@@ -22,6 +27,7 @@ preinit_set_mac_address() {
 		base_mac=$(cat /sys/class/net/eth0/address)
 		ip link set dev eth0 address $(macaddr_add "$base_mac" +2)
 		ip link set dev eth1 address $(macaddr_add "$base_mac" +3)
+		;;
```

Comment:
This fix should be sent in separate patch. This will also have it merged faster than the whole device support PR.

Reply:
Thanks for advice.

Reply:
Do you want to send it? Otherwise I would do it ...

Reply:
Just realized that this seems to be valid syntax for the _last_ case element. Still, I would change it to account for later reorder etc ...